pipeline/db/db.go

```diff
@@ -665,13 +665,13 @@ func GetAltApps(appID string) ([]AltApp, error) {
 	return result, nil
 }
 
-func anyPatternFormat(arr *[]string) {
+func arrRegexAnyFormat(arr *[]string) {
 	for i, v := range *arr {
 		(*arr)[i] = "%" + v + "%"
 	}
 }
 
-func startsWithPatternFormat(arr *[]string) {
+func startsWithRegexFormat(arr *[]string) {
```

Comment:
this is essentially the same. implies that this function is checking if something fits in a specific format